lib/dal/src/workspace_snapshot/node_weight/traits.rs

```diff
@@ -23,10 +27,162 @@ pub type CorrectTransformsResult<T> = Result<T, CorrectTransformsError>;
 pub trait CorrectTransforms {
     fn correct_transforms(
         &self,
-        _workspace_snapshot_graph: &WorkspaceSnapshotGraphV2,
+        _workspace_snapshot_graph: &WorkspaceSnapshotGraphV3,
         updates: Vec<Update>,
         _from_different_change_set: bool,
     ) -> CorrectTransformsResult<Vec<Update>> {
         Ok(updates)
     }
 }
+
+// XXX: Should this actually require Sized? rustc is very unhappy about CorrectExclusiveOutgoingEdge without it.
```

Comment:
I believe the higher-ranked trait bound on CorrectExclusiveOutgoingEdge has an implicit `Sized` bound, and we could maybe change it to `?Sized` to get rid of the `Sized` requirement here

Reply:
I don't know that anything really needs to change. I was just surprised by the error I was getting without the explicit trait bound here.